src/FondOfSpryker/Zed/ThirtyFiveUp/Communication/Plugin/Sales/UpdateThirtyFiveUpOrderAfterPostSaveOrderExpanderPlugin.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace FondOfSpryker\Zed\ThirtyFiveUp\Communication\Plugin\Sales;
+
+use Generated\Shared\Transfer\QuoteTransfer;
+use Generated\Shared\Transfer\SaveOrderTransfer;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+use Spryker\Zed\SalesExtension\Dependency\Plugin\OrderPostSavePluginInterface;
+
+/**
+ * Class UpdateThirtyFiveUpOrderAfterPostSaveOrderExpanderPlugin
+ *
+ * @package FondOfSpryker\Zed\ThirtyFiveUp\Communication\Plugin\Sales
+ *
+ * @method \FondOfSpryker\Zed\ThirtyFiveUp\Business\ThirtyFiveUpFacadeInterface getFacade()
+ * @method \FondOfSpryker\Zed\ThirtyFiveUp\ThirtyFiveUpConfig getConfig()
+ */
+class UpdateThirtyFiveUpOrderAfterPostSaveOrderExpanderPlugin extends AbstractPlugin implements OrderPostSavePluginInterface
```

Comment:
The name doesn't feel right here, there is no "expand" function and it's super verbose.. I need to read it two to three times to barely understand what it does